translations/ui-bulk-edit/en_GB.json

```diff
@@ -193,5 +193,7 @@
     "uploaderSubTitle.holdings.HRID": "Select a file with holdings HRIDs",
     "layer.options.holdings.temporaryLocation": "Temporary holdings location",
     "layer.options.holdings.permanentLocation": "Permanent holdings location",
-    "options.placeholder": "Select option"
-}
\ No newline at end of file
+    "options.placeholder": "Select option",
+    "list.filters.logs": "Logs",
+    "logs.view": "Bulk edit - Can view logs"
+}
```

Comment:
Aren't the translations other than en.json updated by the bot?

Reply:
They update by the bot, but it will take some time. We have practice to update en jsons to see translations on test env